### PR TITLE
(feature: create-carts): Initialization of carts service

### PR DESCRIPTION
**Pull request by juancdelgadosornes**
- Create carts-service main structure
- Create dockerfile for prd and dev environment
- Create make file

### CODE REVIEW

File: src/carts/Dockerfile.dev [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+# Installing dependencies
+COPY go.mod go.sum ./
+RUN go mod download
+
+# Copying all the files
+COPY . .
+
+# Starting our application
+CMD ["go", "run", "src/app/main.go"]
```
Comment (peref-adsk) [inline src/carts/Dockerfile.dev]: this will not work for hot reload

**Review by peref-adsk — CHANGES_REQUESTED** [review]

File: src/carts/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    container_name: carts
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    environment:
+      - PORT=9200
+    ports:
+      - 9200:9200
+    volumes:
+      - .:/app-carts
```
Comment (peref-adsk) [inline src/carts/docker-compose.yaml]: on the Dockerfile.dev the workdir is /app instead of /app-carts

**Review by peref-adsk — CHANGES_REQUESTED** [review]

File: src/carts/docker-compose.yaml (hunk 2 above)
Comment (juancdelgadosornes) [reply src/carts/docker-compose.yaml]: I updated the volume

**Review by peref-adsk — APPROVED** [review]
LGTM

**Pull request merged**